src/NLog/Config/ConfigurationItemFactory.cs

```diff
@@ -427,18 +428,21 @@ private static ConfigurationItemFactory BuildDefaultFactory()
                 var allAssemblies = LogFactory.CurrentAppDomain.GetAssemblies();
                 foreach (var assembly in allAssemblies)
                 {
-                    if (assembly.FullName.StartsWith("NLog.", StringComparison.OrdinalIgnoreCase))
+                    if (assembly.FullName.StartsWith("NLog", StringComparison.OrdinalIgnoreCase))
```

Comment:
This is less strict? This would match nlogger.dll?

Reply:
It matches the current: `Directory.GetFiles(assemblyLocation, "NLog*.dll")`

It is the hidden assembly logic, that is more strict.

Reply:
Though I don't mind reverting it back to `NLog.` (But then we should consider also fixing the `GetFiles`-call

Reply:
Changing the getfiles is backwards compatible, isn't? Would prefer that one.

Reply:
Nope it will suddenly not load `NLogSplunkTarget.dll` (Made Up)

Reply:
But you have convinced me, matching on filename is different from matching on namespace. Have added the dot back, so it remains strict.

Reply:
But the filter start with "nlog." was already there?

Reply:
Yes and now it is back again :)

Reply:
:)